include/junction.hpp

```diff
@@ -52,4 +63,9 @@ inline bool operator<(const junction & lhs, const junction & rhs)
             : rhs.get_mate1() < lhs.get_mate1()
                 ? false
                 : lhs.get_mate2() < rhs.get_mate2();
-}
\ No newline at end of file
+}
+
+inline bool operator==(const junction & lhs, const junction & rhs) // ! Equality without read name!
```

Comment:
Why here without read name?

Reply:
Because a junction can be supported by different reads, but they are still the same junction.
Thats why I have also included a counter for how many reads support a junction.
@eldariont is that a correct assumption?